### DISCUSSION TITLE
Wine Staging

### DISCUSSION THREAD

**Discussion by trymeouteh** [Kron4ek/Wine-Builds#69] [Q&A] [answered]
Are wine staging build the same as wine staging from WineHQ? Or are different staging patches applied?

**Comment by Kron4ek** [comment, marked as answer]
I apply all available staging patches (with `patchinstall.sh --all`) to my Staging builds. And from what I see in the source code of the latest WineHQ staging build, they apply all the patches too, but I can't say for sure.
